lib/amo-client.js

```diff
@@ -116,17 +116,24 @@ Client.prototype.waitForSignedAddon = function(statusUrl, opt) {
         lastStatusResponse = data;
 
         var failedValidation = !data.valid;
-        // The add-on passes validation and all files have been created.
-        var signedAndReady = (data.valid && data.reviewed && data.files &&
-                              data.files.length > 0);
+        // The add-on passed validation and all files have been created.
+        var signedAndReady = (data.valid && data.active && data.reviewed &&
+                              data.files && data.files.length > 0);
+        // The add-on is valid but requires a manual review before it can be signed.
+        var requiresManualReview = data.valid && !data.active && !data.reviewed;
```

Comment:
The API should probably return `automated_signing` instead of making you do this.